lib/elixir_conf_africa/cart.ex

```diff
@@ -0,0 +1,86 @@
+defmodule ElixirConfAfrica.Cart do
+  @moduledoc """
+  The cart module
+  """
+
+  alias ElixirConfAfrica.TicketTypes
+
+  @doc """
+  Returns the ids of all the cart items in a list
+
+
+
+  """
+  def cart_list_ids(cart) do
+    cart |> Enum.map(fn cart_item -> cart_item.id end)
+  end
+
+  @doc """
+  Increases the quantity of a cart item given the cart item and the quantity to increase by
+
+  """
+
+  def increase_cart_item_quantity(cart_item, quantity) do
+    cart_item
+    |> Map.put(:quantity, quantity)
+  end
+
+  @doc """
+  Adds a cart item to the cart given the cart and the cart item id to add
+
+  """
+  def add_to_cart(cart, cart_item_id) do
+    cart_list_ids = cart_list_ids(cart)
+
+    case Enum.member?(cart_list_ids, cart_item_id) do
```

Comment:
A Case statement is overkill here. You can just use an if

Reply:
True , since it is either true or false